core/base/src/TApplication.cxx

```diff
@@ -1052,11 +1052,30 @@ void TApplication::Help(const char *line)
       gInterpreter->ProcessLine(line);
       Printf("\nROOT special commands.");
       Printf("==============================================================================");
-      Printf("   .pwd                : show current directory, pad and style");
-      Printf("   .ls                 : list contents of current directory");
-      Printf("   .which [file]       : shows path of macro file");
+      Printf("   .credits            : Show credits");
+      Printf("   .demo               : Launch GUI demo");
       Printf("   .help Class         : opens the reference guide for that class (or .?)");
       Printf("   .help Class::Member : opens the reference guide for function/member (or .?)");
+      Printf("   .license            : Show license");
+      Printf("   .ls                 : list contents of current TDirectory");
+      Printf("   .pwd                : show current TDirectory, pad and style");
+      Printf("   .quit (or .exit)    : quit ROOT (long form of .q)");
+      Printf("   .R [user@]host[:dir] [-l user] [-d dbg] [script]\n"
+             "                       : Launch process in a remote host");
+      Printf("   .qqq                : quit ROOT - mandatory");
+      Printf("   .qqqqq              : exit process immediately");
+      Printf("   .qqqqqqq            : abort process");
+      Printf("   .which [file]       : shows path of macro file");
+      Printf("   .![OS_command]      : executes OS-specific shell command");
+      Printf("\nROOT terminal keyboard shortcuts.");
+      Printf("==============================================================================");
+      Printf("   CTRL+C              : send kSigInt interrupt signal");
+      Printf("   CTRL+D              : exit ROOT");
+      Printf("   CTRL+Del            : remove next word");
+      Printf("   CTRL+L              : clear screen");
+      Printf("   CTRL+Left           : jump to previous word");
+      Printf("   CTRL+Right          : jump to next word");
```

Comment:
Maybe https://apple.stackexchange.com/questions/18043/how-can-i-make-ctrlright-left-arrow-stop-changing-desktops-in-lion

Reply:
@jalopezg-r00t  Maybe the keyboard shortcuts should be OS-dependent, making them CTRL+ALT in Mac so that they do not collide with the system's ones? Or we specify they are Unix-only?

Reply:
These shortcuts should work as long as they are delivered to the terminal emulator (instead of being intercepted by something else).  IMHO, making them OS-dependent would only create more confusion.

Reply:
Agreed but the help should may be mention it ? Mac is an important platform and "move left/right a space" are the default. So by default  this does not work on Mac.

Reply:
I added a footnote-mention.